### PR TITLE
Address follow-up feedback from #1934

### LINKED ISSUE #2026: Address feedback from Mono CI PR
See https://github.com/dotnet/runtime/pull/1934

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/runtime/issues/2026

cc @akoeplinger

### CODE REVIEW

File: eng/pipelines/runtime.yml [hunk 1]
```diff
@@ -217,13 +217,15 @@ jobs:
     # - Windows_NT_arm
     # - Windows_NT_arm64
     jobParameters:
-      condition: >-
+      condition: >-
```
Comment (akoeplinger) [inline eng/pipelines/runtime.yml]:
```suggestion
      condition: >-
```

File: eng/pipelines/runtime.yml [hunk 2]
```diff
@@ -241,6 +243,12 @@ jobs:
     # - Windows_NT_x86
     # - Windows_NT_arm
     # - Windows_NT_arm64
+    jobParameters:
+      condition: >-
```
Comment (akoeplinger) [inline eng/pipelines/runtime.yml]:
```suggestion
      condition: >-
```

File: eng/pipelines/runtime.yml [hunk 3]
```diff
@@ -504,6 +513,11 @@ jobs:
       liveRuntimeBuildConfig: release
       dependsOnTestBuildConfiguration: ${{ variables.debugOnPrReleaseOnRolling }}
       dependsOnTestArchitecture: x64
+      condition: >-
```
Comment (akoeplinger) [inline eng/pipelines/runtime.yml]:
```suggestion
      condition: >-
```

**Review by akoeplinger — COMMENTED** [review]
Found some whitespace nits :)

**Review by stephentoub — APPROVED** [review]

**Review by safern — APPROVED** [review]
Thanks for following up.

**Pull request merged**